sdk/core/core-amqp/src/cbs.ts

```diff
@@ -258,9 +259,9 @@ export class CbsClient {
         await cbsLink!.close();
         logger.verbose("[%s] Successfully closed the cbs session.", this.connection.id);
       }
-    } catch (err: any) {
+    } catch (err) {
       const msg = `An error occurred while closing the cbs link: ${
-        err.stack || JSON.stringify(err)
+        isError(err) ? err.stack || JSON.stringify(err) : JSON.stringify(err)
```

Comment:
is `err.stack` not guaranteed to exist? I suppose not. Maybe this?

```suggestion
        (isError(err) && err.stack) ? err.stack : JSON.stringify(err)
```